packages/docs/timetable-planner/README.md

```diff
@@ -48,169 +51,109 @@ When preferences are made, those times will not be changed.
 
 ![lock1](./lock1.png)
 
-after some preferences made:
+after a course is locked and a preference is being made:
 
 ![lock2](./lock2.png)
 
+## Roadmap for the Timetable Algorithm
+<!---
+- Introduce what the timetable algorithm is for
+- Tell a story about how the algorithm evolves
+- Have a heading for each optimization in the roadmap
+- Start with base conflict check -> invalid times -> idle time max/min -> locked courses
+--->
+Timetable algorithm takes in a set of course names and transform into a list of timetables. Then the user can optimize the timetable to fit their preference.
 
-## Example Usage:
-
-### Future Implementation:
-
-Given a set of course codes, the timetable planner outputs the schedule available for the set.
-
-If no such schedule available, outputs "inValid".
-
-//Setting up preference are still work in progress
-
-#### Input:
-
-```json
-#provided two courses with two section for simplicity
-{
-    "CourseCode": {
-        "CSC108H5F2019":{
-            "L0101":[{
-                "MONDAY":[32400, 36000],
-                "WEDNESDAY":[32400, 36000],
-                "FRIDAY":[32400, 36000],
-                "size":160,
-                "enrolment":0
-
-            }],
-            "L0107":[{
-                "WEDNESDAY":[64800, 75600],
-                "size":160,
-                "enrolment":0
-                }]
-        },
-        "CSC318H5F2019":{
-            "L0101":[{
-                "TUESDAY":[68400, 75600],
-                "WEDNESDAY":[68400, 75600],
-                "size": 60,
-                "enrolment": 0
-            }],
-            "T0107":[{
-                "THURSDAY":[68400, 75600],
-                "size": 60,
-                "enrolment": 0
-                }]
-        },
-        
-}
+The algorithm starts out basic, it checks if the sections given have conflict to each other. If there is no conflict, the algorithm will return "valid timetable", or if there is a conflict between any section, it will return "invalid timetable".
 
-```
+Based on the validity checking, the algorithm starts to be able to add constraints to the timetable.
```

Comment:
After checking for basic validity, we can optimize our timetable by adding constraints to it.